enterprise/static/enterprise/sass/_base.scss

```diff
@@ -206,27 +162,27 @@ body,
   }
 
   .success {
-    background-color: #ecfaec;
-    border-color: #b9edb9;
-    color: #3c763d;
+    background-color: $success-100;
+    border-color: $success-200;
+    color: $success-500;
   }
 
   .info {
-    background-color: #f2f8fb;
-    border-color: #cce3f0;
-    color: #31708f;
+    background-color: $info-100;
+    border-color: $info-200;
```

Comment:
Should these `.info`, `.success`, `.warning`, etc. classes have a border defined? The new theme does not have borders. Alternatively, should we be using these custom classes? Or could we swap them out for similar `.alert .alert-info`, `.alert .alert-success`, etc. which I imagine would apply the appropriate styles from the theme instead of these custom classes?